includes/definitions/discovery/eaton-mgeups.yaml

```diff
@@ -0,0 +1,234 @@
+mib: MG-SNMP-UPS-MIB
+modules:
+    sensors:
+        power:
+            data:
+                -
+                    oid: upsmgOutput
```

Comment:
For all of these, you've used a high level folder / OID when you should be using the Table. See this pic:

![image](https://user-images.githubusercontent.com/3941142/46667316-3dc13900-cbc1-11e8-9452-121f72470dde.png)

So use upsmgOutputPhaseTable in this one example but the rest also need to be updated.

Reply:
It will always do a walk but if you will be re-using data within the folder OID then it will be fine to use it as we cache the original walk so long as you aren't walking a lot of data.

It's a trade off between walking too much data to cache and not caching enough and having to perform multiple walks. Neither is perfect just use the best option.